install/kubernetes/cilium/files/cilium-envoy/configmap/bootstrap-config.json

```diff
@@ -388,11 +388,20 @@
       }
     ]
   },
+  "application_log_config": {
+    "log_format": {
+{{- if .Values.envoy.log.format_json }}
+      "json_format": {{ .Values.envoy.log.format_json | toJson }}
+{{- else }}
+      "text_format": "{{ .Values.envoy.log.format }}"
+{{- end }}
+    }
+  },
   "admin": {
     "address": {
       "pipe": {
         "path": "/var/run/cilium/envoy/sockets/admin.sock"
       }
     }
-  }
+  },
```

Comment:
nit: this comma is extra :-)

Reply:
Good catch, thanks! updated in https://github.com/cilium/cilium/commit/fd54b009cd483ff1ad1c68ff11da4fee4b53d779